src/instructlab/cli/process/attach.py

```diff
@@ -0,0 +1,24 @@
+# Third Party
+import click
+
+# First Party
+from instructlab import clickext
+
+
+@click.command(name="attach")
+@clickext.display_params
+@click.option("--uuid", type=click.STRING, help="PID of the process to attach to")
+@click.option("--latest", is_flag=True, help="attach to the latest process")
+def attach(uuid, latest):
+    # First Party
+    from instructlab.process.process import attach_process, get_latest_process
```

Comment:
Why not make this a global import?

Reply:
I'm fine with either, I just want the motivation to make sense. When is a local import preferred over a global?

Reply:
generally speaking, local imports are typically used to guard specific things from being imported unless explicitly needed aka: torch. With this, nothing in process.py seems to be something we'd want to guard in this way, I can move the imports to be global

Reply:
SGTM!